src/cares_wrap.cc

```diff
@@ -36,21 +36,21 @@
 #include <unordered_set>
 
 #ifdef __POSIX__
-# include <netdb.h>
+  #include <netdb.h>
 #endif  // __POSIX__
 
 #if defined(__ANDROID__) || \
     defined(__MINGW32__) || \
     defined(__OpenBSD__) || \
     defined(_MSC_VER)
 
-# include <nameser.h>
+  #include <nameser.h>
 #else
-# include <arpa/nameser.h>
+  #include <arpa/nameser.h>
 #endif
 
 #if defined(__OpenBSD__)
-# define AI_V4MAPPED 0
+  #define AI_V4MAPPED 0
 #endif
```

Comment:
Getting closer :-) ... the indent here should come after the `#` ... so

```c
#if defined(__OpenBSD__)
# define AI_V4MAPPED 0
#endif 
```

is the correct style.

the linter needs to be updated to check for the indenting after the `#` and should complain if the `#` itself is indented.